frontend/src/views/Settings/OrgSettingsPage/OrgSettingsPage.tsx

```diff
@@ -293,41 +301,34 @@ export const OrgSettingsPage = () => {
-							</p>
-							<p className="mt-4 text-md text-gray-400">
-								As soon as you delete an organization, you will
-								not be able to undo it. This will immediately
-								remove all organization members and cancel your
-								subscription. If you still want to do that,
-								please enter the name of the organization below.
-							</p>
-							<div className="max-h-28 w-full max-w-xl mr-auto mt-8 max-w-xl">
-								<InputField
-									label="Organization to be Deleted"
-									onChangeHandler={
-										setWorkspaceToBeDeletedName
-									}
-									type="varName"
-									value={workspaceToBeDeletedName}
-									placeholder=""
-									isRequired
-								/>
-							</div>
-							<button
-								type="button"
-								className="mt-6 w-full max-w-xl inline-flex justify-center rounded-md border border-transparent bg-gray-800 px-4 py-2.5 text-sm font-medium text-gray-400 hover:bg-red hover:text-white hover:font-bold hover:text-semibold duration-200 focus:outline-none focus-visible:ring-2 focus-visible:ring-blue-500 focus-visible:ring-offset-2"
-								onClick={executeDeletingWorkspace}
-							>
-								Delete Project
-							</button>
-							<p className="mt-0.5 ml-1 text-xs text-gray-500">
-								Note: You can only delete a project in case you
-								have more than one.
-							</p>
-						</div> */}
+        <div className="mb-6 mt-4 flex w-full max-w-6xl flex-col items-start border-l border-red pb-4 pl-6 pt-2">
+          <p className="text-xl font-bold text-red">Danger Zone</p>
+          <p className="text-md mt-4 text-gray-400">
+            As soon as you delete an organization, you will not be able to undo it. This will
+            immediately remove all organization members and cancel your subscription. If you still
+            want to do that, please enter the name of the organization below.
+          </p>
+          <div className="mr-auto mt-8 max-h-28 w-full max-w-xl">
+            <p className='text-md my-2 text-gray-400'>Organization to be deleted</p>
+            <Input
+              placeholder="Organization name"
+              onChange={(e) => setOrganizationToBeDeletedName(e.target.value)}
+              type="varName"
+              value={organizationToBeDeletedName}
+              isRequired
+            />
+          </div>
+          <button
+            type="button"
+            className="hover:text-semibold mt-6 inline-flex w-full max-w-xl justify-center rounded-md border border-transparent bg-gray-800 px-4 py-2.5 text-sm font-medium text-gray-400 duration-200 hover:bg-red hover:font-bold hover:text-white focus:outline-none focus-visible:ring-2 focus-visible:ring-blue-500 focus-visible:ring-offset-2"
+            onClick={executeDeletingOrganization}
+          >
+            Delete Organization
+          </button>
+          <p className="mt-0.5 ml-1 text-xs text-gray-500">
+            Note: You can only delete a organization in case you have more than one.
```

Comment:
TODO: Remove this now that we've discussed that it is ok for users to not belong to any organization (e.g. if they get kicked out of the only one that they were initially invited to).